src/GuppyDataExplorer/TierAccessSelector/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@gen3/ui-component/dist/components/Button';
+import './TierAccessSelector.css';
+
+/**
+ * Tier access selector has 3 options: 'Data with Access', 'Data without Access', and 'All Data'
+ * By default "All Data" is selected
+ */
+class TierAccessSelector extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: 'all-data',
+      toggled: false,
+    };
+  }
+
+  handleSelectorChange = (e) => {
+    this.props.onSelectorChange(e.target.value);
+    this.setState({ selected: e.target.value });
+  };
+
+  handleToggle = () => {
+    this.setState(prevState => ({ toggled: !prevState.toggled }));
+  }
+
+  render() {
+    return (
+      <div className='tier-access-selector'>
+        <h4 // eslint-disable-line jsx-a11y/no-noninteractive-element-interactions
```

Comment:
also don't think we should disable it - it just needs a `role` and `tabIndex`
good for accessibility :)

Reply:
`tabIndex` work for `div` but not `h4` according to this rules https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md .  😂

Reply:
can we make it a div then? :)